app/static/css/vendor/gentelella.css

```diff
@@ -5185,3 +5185,6 @@ ul.notifications {
   width: 0;
   }
 }
+#basic-addon3{
```

Comment:
Do not make any changes in this file or any other file inside the vendor directory.